config/packages/doctrine.yaml

```diff
@@ -27,6 +27,12 @@ doctrine:
             Domain:
                 is_bundle: false
                 type: yml
-                dir: '%kernel.project_dir%/config/mapping/orm/query'
+                dir: '%kernel.project_dir%/config/mapping/orm/domain'
```

Comment:
we can remove domain from orm config as is not one of domain responsibilities

Reply:
What do you propose? deleting the folder domain and moving the mapping of the VO to infrastructure?

Reply:
Yes, exactly that.

Reply:
If I do that. I will need to rename the mappings files to include Domain/Infrastructure when necessary.

I'm going to do a commit with that change and we'll review it.